api/v1/openlibertyapplication_types.go

```diff
@@ -54,98 +54,102 @@ type OpenLibertyApplicationSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:order=8,type=spec,displayName="Expose",xDescriptors="urn:alm:descriptor:com.tectonic.ui:booleanSwitch"
 	Expose *bool `json:"expose,omitempty"`
 
-	// Enable management of LTPA key sharing amongst Liberty containers. Defaults to false.
+	// Enable management of password encryption key sharing amongst Liberty containers. Defaults to true.
```

Comment:
- We should set this value to false by default. That'll allow the apps that want to use this new feature to opt-in, as opposed to forcing all apps due to one app owner creating the password key secret.
- The display name `Manage LTPA` should be changed.